### DISCUSSION TITLE
importing the microstructure created neper in the ABAQUS

### DISCUSSION THREAD

**Discussion by kumaranvenk** [neperfepx/neper#831] [Q&A] [answered]
I have created .tess file and .msh file in neper. I want to know how to import it into ABAQUS and what are the steps to perform crack propagation in the microstrucuture using ABAQUS

**Comment by rquey** [comment, marked as answer]
You can write the mesh at Abaqus inp format using `-format inp` in -M.
re: crack propagation, this is more an Abaqus question.

**Reply by kumaranvenk** [reply]
is it possible to get inp format in -T  ??

**Reply by rquey** [reply]
No.
